features/read.feature

```diff
@@ -26,3 +26,12 @@ Feature: Read Cache Time
   Scenario: Read non-existing Cache Time resource
     When I GET "/v1/cache-times/5d41402abc4b2a76b9719d911017c592"
     Then the HTTP status code should be "404"
+
+  Scenario: Read Cache Time resource with invalid ID format
+    When I GET "/v1/cache-times/INVALID-ID"
+    Then I should receive the following JSON response with status "400":
+            """
+            {
+                "error": "validation errors: [id should be 32 characters in length id is not lowercase id is not a valid hexadecimal]"
```

Comment:
Is there a way to make the error message a bit more readable? Even just separating the different errors with a comma would make it easier to read. In reality, given that this is JSON, an array of strings would be ideal (each string being a different error), but if that's too messy or complicated to implement, then clearly separating the different errors should be enough.